src/components/VersionChooser/index.tsx

```diff
@@ -132,6 +155,19 @@ const mapStateToProps = (
   };
 };
 
+const ConnectedVersionChooser = connect(mapStateToProps)(VersionChooserBase);
+
+export const VersionChooserWithoutRouter = ConnectedVersionChooser as React.ComponentType<
+  PublicProps &
+    Partial<
+      DefaultProps & {
+        match: {
```

Comment:
It would be confusing to understand why this type is declared because it's not necessary for normal functionality. A comment would help.